ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/aaa.yml

```diff
@@ -29,6 +29,41 @@ radius_servers:
     vrf: default
     key: 071B245F5A
 
+### Radius Server
+# New data model to replace "radius_servers"
+radius_server:
+  attribute_32_include_in_access_req:
+    hostname: true
+  dynamic_authorization:
+    port: 1700
+    tls_ssl_profile: SSL_PROFILE
+  hosts:
```

Comment:
Please avoid duplicate hosts with vars for old data model. Just use 10.10.11.x here so make it simple.

Reply:
Fixed